code/modules/admin/admin_verbs.dm

```diff
@@ -364,6 +364,8 @@ var/list/admin_verbs_hideable = list(
 	if(holder)
 		holder.check_antagonists()
 		log_admin("[key_name(usr)] checked antagonists.")	//for tsar~
+		if(isliving(usr))
```

Comment:
Should make that a inverse observer check, other wise admins can check antags in lobby and late join without a message to admins.